src/main/java/bio/terra/service/dataset/DatasetRequestValidator.java

```diff
@@ -381,13 +386,16 @@ private void validateAsset(AssetModel asset, Errors errors, SchemaValidationCont
   private void validateSchema(DatasetSpecificationModel schema, Errors errors) {
     SchemaValidationContext context = new SchemaValidationContext();
     List<TableModel> tables = schema.getTables();
-    if (tables != null) {
+    if (tables != null && !tables.isEmpty()) {
```

Comment:
The condition here is flipped from the above on line 188. We should have consistent conditions within a function for similar operations. I prefer the style on line 188 because it involves no negation.